README.markdown

```diff
@@ -497,8 +497,9 @@ Pass the lat/lng as a string, array or LatLng instance:
 
 ```ruby
 res=Geokit::Geocoders::GoogleGeocoder.reverse_geocode "37.791821,-122.394679"
-=> #<Geokit::GeoLoc:0x558ed0 ...
-res.full_address "101-115 Main St, San Francisco, CA 94105, USA"
```

Comment:
When first reading this, I thought I should pass the address as an argument to `full_address`, which is not true